Automattic/woocommerce-payments
# Set mandatory support phone

Fixes #9556 

#### Changes proposed in this Pull Request

The support phone number field on the Payment settings page defaults to empty for newly onboarded accounts. In this PR the support phone is set to mandatory, so the user is forced to set the phone number.










#### Testing instructions





* onboard new account
* go to `WooCommerce -> Settings` page
* an error should be displayed if the phone field is empty

<img width="523" alt="Screenshot 2024-11-20 at 13 29 23" src="https://github.com/user-attachments/assets/9a2b626d-16b7-4cbd-9b42-3bd5f7a66176">

* the "Save changes" button should NOT be active until we have the support phone number is set


-------------------


**Post merge**
